src/main/java/com/devonfw/quarkus/ordermanagement/logic/UcManageOrder.java

```diff
@@ -0,0 +1,89 @@
+package com.devonfw.quarkus.ordermanagement.logic;
+
+import com.devonfw.quarkus.general.restclient.product.ProductsRestClient;
+import com.devonfw.quarkus.general.restclient.product.models.ProductDto;
+import com.devonfw.quarkus.ordermanagement.domain.model.ItemEntity;
+import com.devonfw.quarkus.ordermanagement.domain.model.OrderEntity;
+import com.devonfw.quarkus.ordermanagement.domain.model.OrderStatus;
+import com.devonfw.quarkus.ordermanagement.domain.repo.ItemRepository;
+import com.devonfw.quarkus.ordermanagement.domain.repo.OrderRepository;
+import com.devonfw.quarkus.ordermanagement.service.v1.model.NewOrderDto;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.transaction.Transactional;
+import javax.ws.rs.core.Response;
+import java.math.BigDecimal;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Named
```

Comment:
Shouldn't we use `@ApplicationScoped` instead of `@Named` as described in the [dependency injection guide](https://github.com/devonfw/devon4j/blob/master/documentation/guide-dependency-injection.asciidoc#example-bean)

Reply:
You are right. I think we need both